src/components/templates/PartyMap/components/Map/Map.scss

```diff
@@ -117,6 +117,7 @@ $border-radius: 28px;
       width: 100%;
       height: 100%;
       border-radius: 100%;
+      z-index: 12;
```

Comment:
Given this is in a scss file, it can use the `z()` helper with a properly named layer.